src/infra/flow-system/inmem/Cargo.toml

```diff
@@ -27,6 +27,7 @@ kamu-core = { workspace = true }
 kamu-task-system = { workspace = true }
 kamu-task-system-services = { workspace = true }
 kamu-flow-system = { workspace = true }
+kamu-flow-system-services = { workspace = true }
```

Comment:
There should not be this dependency.
Repositories should be unaware of service layer existence.
Service layer may not link particular repository implementations as well, with the exception of importing this in-memory repository only as dev-dependency for tests.

Reply:
Eagle eyes -- Done